org.mwc.debrief.pepys/src/main/java/org/mwc/debrief/pepys/model/bean/Comments.java

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+ * Debrief - the Open Source Maritime Analysis Application
+ * http://debrief.info
+ *
+ * (C) 2000-2020, Deep Blue C Technology Ltd
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the Eclipse Public License v1.0
+ * (http://www.eclipse.org/legal/epl-v10.html)
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *******************************************************************************/
+
+package org.mwc.debrief.pepys.model.bean;
+
+import java.util.Date;
+
+public class Comments implements AbstractBean, FilterableBean, TreeStructurable{
```

Comment:
@saulhidalgoaular - since this represents a single row, it should be called `Comment` in my opinion.

Reply:
Good morning. Yes. I totally agree that. I would also suggest the change of the names in the database tables to singular because they represent entities.

The thing is that I am using reflection to get the name of the table, from the class name. Changing it would mean the change of how I would retrieve the table now.

Reply:
> using reflection to get the name of the table

Aah yes, there are two conventions for table names. We took the decision a year ago to use plural names.

So, yes - we should keep `Comments` if we are going to use reflection.  But alternatively, is it an option to use a new annotation to store the @TableName?

Then we could have `Comment` in Java for a single comment instance, with the knowledge that the data is coming from the `Comments` table?

(this isn't very important to me, I'm just offering a suggestion).